packages/wpcom-checkout/src/get-country-postal-code-support.ts

```diff
@@ -1,15 +1,35 @@
 import type { CountryListItem } from './types';
 
 /**
- * Returns true if postal codes are supported on the specified country code,
- * or false otherwise.
+ * Returns true if postal codes are supported on the specified country code.
+ *
+ * NOTE: Will return false if the countries list is empty or if the country
+ * code is not in the list! Always check that the countries list has loaded
+ * before calling this.
  */
 export function getCountryPostalCodeSupport(
 	countries: CountryListItem[],
 	countryCode: string
 ): boolean {
-	return (
-		countries.find( ( country ) => country.code === countryCode.toUpperCase() )?.has_postal_codes ??
-		false
+	if ( ! countryCode ) {
+		return false;
+	}
+	if ( ! countries?.length ) {
+		// eslint-disable-next-line no-console
+		console.error(
```

Comment:
Would it be better to use debug here and for the console.warn on line 29?

Reply:
I don't think so, because if either of these show up, it could lead to very unexpected behavior and I want anyone who sees the log to notice, even if they don't have debugging on. Really, IMO they should be thrown errors but that feels too risky a change to me right now.